Makefile

```diff
@@ -185,8 +185,9 @@ set -e ;\
 TMP_DIR=$$(mktemp -d) ;\
 cd $$TMP_DIR ;\
 go mod init tmp ;\
-echo "Downloading $(2)" ;\
+echo "Downloading $(2) to $(PROJECT_DIR)" ;\
 GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
+GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
```

Comment:
i had one of the tool relying on the executable to be present in the bin/ subfolder of my workspace, which required a `go install`  on my machine (go go1.18.2)

Reply:
After you rebase on master this should be unnecessary (#984)